### PR TITLE
Update Install docs

### PR DESCRIPTION
**Pull request by ravimotha**
added extra information if you want to run windows on mac

### CODE REVIEW

**Comment by marcemarc** [comment]
Hi Ravi!

I don't know an awful lot about running Umbraco on a Mac, (although I use Bootcamp, I believe).

Anyway it's super useful I think to put information together regarding this.

The place on the index page that you have updated, currently links through to this page: https://our.umbraco.org/documentation/Getting-Started/Setup/Install/running-umbraco-on-a-mac

Would you say the information on that page is correct?

I'm thinking instead of adding the specifics on the install index page, I'm thinking we keep this 'running umbraco on a mac' page up-to-date...

but I do think we need to tweak the text on the index, page, to make it clear **there is** more information on 'doing so' - if you click the link...

what do you think?

cheers

Marc

**Comment by ravimotha** [comment]
Interesting thought.. 
The actual technical stuff on the page is actually fine. but it does feel like the text is missing and somewhat thin, and doesn't indicate there is more text. I think maybe thats why i thought was a simple intro to do that..

**Comment by ravimotha** [comment]
Maybe the edit needs to be something like this
 Click on the provided link to find out more information and tools you can use to run windows on mac 
 rather than..
+You can use tool like [Windows Parallels](https://www.parallels.com/uk/products/desktop/) or [Virtual Box](https://www.virtualbox.org/) to run windows in your OS environment or Use [Bootcamp](https://support.apple.com/en-gb/boot-camp) to set your mac to boot into Windows

**Comment by marcemarc** [comment]
Hmm, yes it's giving the scent to the reader that there is more info on a dedicated page...

... perhaps all we need here is to make this text:

"but it's really easy to get Windows up and running on your Mac. "

in the existing paragraph, a link to the page with the info?

or perhaps with a 'Find out How' call to action type link at the end...

**Comment by ravimotha** [comment]
I like the find out how type option

Click on the link  [some text] to find out ore about how to do this and the
tools available to you

**Comment by marcemarc** [comment]
Hi @ravimotha did we decide to update this? and do you have a PR, with the adjusted text readyyyyyy to rollll?

**Comment by ravimotha** [comment]
Well I put my comment regarding 
Click on the link  [some text] to find out more about how to do this and the tools available to you


but we didn't go any further.. ultimately I think its okay. but my edit reads better to me (obviously biased)

**Comment by ravimotha** [comment]
oh and where i put [some text ] is supposed to be the link to the more information page

**Comment by marcemarc** [comment]
@ravimotha Yes, I think your text is good to go, if you update the PR I'll merge it in!

**Comment by ravimotha** [comment]
Commit updated ready for your review

File: Getting-Started/Setup/Install/index.md [hunk 1]
```diff
@@ -23,4 +23,4 @@ NuGet is the package manager for the Microsoft development platform, including .
 
 ### [Running Umbraco on a Mac](running-umbraco-on-a-mac.md)
 Umbraco does not run natively on Mac OS but it's really easy to get Windows up and running on your Mac.
-You can use tool like [Windows Parallels](https://www.parallels.com/uk/products/desktop/) or [Virtual Box](https://www.virtualbox.org/) to run windows in your OS environment or Use [Bootcamp](https://support.apple.com/en-gb/boot-camp) to set your mac to boot into Windows
+Click on the link [tools and information on how to run Umbraco on a mac](running-umbraco-on-a-mac.md) to find out ore about how to do this and the tools available to you to find out more about how to do this and the
```
Comment (marcemarc) [inline Getting-Started/Setup/Install/index.md]: eek @ravimotha , it's gone a bit weird in this last sentence...

sorry, it's a saga now!

it's repeating the 'to find out more about how to do this and the' bit at the end :-(

**Comment by ravimotha** [comment]
That’s my fault losing the ability to see age etc 
 I must have pasted the sentence twice by accident 

Will rectify and let you know

**Comment by ravimotha** [comment]
updated and pushed to patch one.. sighs hopefully get this sorted and ten
i can start looking at other things slowly and ramp up once i get a flavour

**Comment by marcemarc** [comment]
woot! thanks @ravimotha it's merged!

I shall request to get a shiny C-Trib badge on your our profile!

**Comment by dampee** [comment]
Welcome to the club Ravi!  :-)

**Comment by ravimotha** [comment]
this is cool.. yes I am a geek.. cheers baby steps  right starts looking
for next changes..

**Pull request merged**